src/HomogeneousCalorimeter_geo.cpp

```diff
@@ -70,23 +70,41 @@ static Ref_t create_detector(Detector& desc, xml::Handle_t handle, SensitiveDete
   // module placement
   xml::Component     plm = detElem.child(_Unicode(placements));
   std::map<int, int> sectorModuleNumbers;
```

Comment:
It looks I can't put sector ID, row and column information into the map container here.